modules/core/src/test/java/org/apache/ignite/testframework/junits/common/GridCommonAbstractTest.java

```diff
@@ -2357,4 +2361,32 @@ protected void assertCountersSame(int partId, boolean withReserveCntr) throws As
             cntr0 = cntr.get2();
         }
     }
+
+    /**
+     * Returns MX bean by specified group name and class.
```

Comment:
Please move method to GridAbstractTest, if it's possible

Reply:
It is possible, but likely makes no sense as GridAbstractTest has only one child (GridSpiAbstractTest) and for now I see no reason that it (or it's children) needs this method